website/static/style.css

```diff
@@ -23,7 +23,11 @@ a {
     text-decoration: none;
 }
 
-
+.link-color{
+    color: blue; 
+    text-decoration: underline;
+    font-family: 'Times New Roman', Times, serif;
```

Comment:
Leaving Note to verify this font matches the rest of the box info text (or that we want it to be different).